packages/ontario-design-system-component-library/src/components/ontario-card/ontario-card.scss

```
@@ -186,7 +190,7 @@ $ontario-card-heading-colours: (
 
 .ontario-card__text-container {
 	.ontario-card__card-type--horizontal & {
-		width: 66.6%;
+		width: 66.66%;
```

Seeing as this is a repeating decimal, using it's fractional form might yield a more consistent result.

```suggestion
		width: math.percentage(2/3);
```

---

Good catch on the use of `math.div`, SASS has been busy changing things haha